cms/static/js/programs/collections/course_runs_collection.js

```diff
@@ -2,19 +2,18 @@ define([
     'backbone',
     'jquery',
     'js/programs/utils/api_config',
-    'js/programs/collections/auto_auth_collection',
```

Comment:
Is this still used anywhere? At a glance, it doesn't seem to be, in which case you should also delete the code at `js/programs/collections/auto_auth_collection`.